Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1403,4 +1403,49 @@ public function isMonolithicPhp()
 
         return 0 === count($kinds[T_INLINE_HTML]) + count($hhvmHashBangs) && 1 === count($kinds[T_OPEN_TAG]) + count($kinds[T_OPEN_TAG_WITH_ECHO]) + count($hhvmOpenTagsWithEcho);
     }
+
+    /**
+     * Clear tokens at given indexes.
+     *
+     * Merges whitespace tokens around the cleared tokens if needed
+     *
+     * @param int[] $indexes
+     */
+    public function clearTokens(array $indexes)
```

Comment:
> > this vs `clearRange`
> 
> from - till

For end user it may be misleading which method to use.
Both are "clear some tokens", and then if you go inside you see that one will also merge some tokens, while the other not.

I would suggest to remove array (so pass _one_ index), and rename the method